### PR TITLE
feat: Adds a smarter default for `in_project` (Ready for Review)

### LINKED ISSUE #116: `in_project` can also be determined by `application:get_application/1`
This is what the internal Elixir stacktrace formatting uses.

https://github.com/elixir-lang/elixir/blob/7e4fbe657dbf9c3e19e3d2bd6c17cc6d724b4710/lib/elixir/lib/exception.ex#L550

### PR DESCRIPTION
**Pull request by isaacsanders**
Fixes #116

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -212,6 +213,17 @@ Produces the failure message
 
 If a sanitizer function throws an exception while running, it will log out a warning and return the string `[CENSORED DUE TO SANITIZER EXCEPTION]`
 
+### `application`
+
+**Default:** `nil`
+
+If you want a smart default for the `in_project`, you can set the `application`
+value. Set this to the name of your application with the following:
```
Comment (nirev) [inline README.md:221]:
```suggestion
Sets the `application` name to generate a smart default for the `in_project` value. Set this to the name of your application with the following:
```

**Comment by nirev** [comment]
Can you add a test case for this in `payload_test.exs` ?

**Comment by isaacsanders** [comment]
Added the test

**Review by nirev — APPROVED** [review]
❤️ 💛 💙 🖤 💜 💚

**Pull request merged**